tests/test_2024_11_26.py

```diff
@@ -0,0 +1,379 @@
+        # 15: else if (16) then (17) else (18)
+        Param(
+            SpecialArgumentID.LOGIC_OP_PARAM_ID,
+            Op.IF_ELSE,
+            encode_argument_value_if(condition=16, success=17, failure=18),
+        ),
+        # 16: (_token == ETH)
+        Param(token_arg_index, Op.EQ, ArgumentValue(eth_limit.address)),
+        # 17: { return _amount <= 1000 }
+        Param(amount_arg_index, Op.LTE, ArgumentValue(eth_limit.limit)),
+        #
+        # 18: else { return false }
+        Param(SpecialArgumentID.PARAM_VALUE_PARAM_ID, Op.RET, ArgumentValue(0)),
+    ]
+
+def test_vote(helpers, accounts, vote_ids_from_env, stranger):
+
+    # misc
+    easy_track = interface.EasyTrack("0xF0211b7660680B49De1A7E9f25C65660F0a13Fea")
+    nor = contracts.node_operators_registry
+    prepare_agent_for_usdc_payment(15_000_000 * (10**6))
+    prepare_agent_for_steth_payment(20_000 * 10**18)
+    EVM_SCRIPT_EXECUTOR = "0xFE5986E06210aC1eCC1aDCafc0cc7f8D63B3F977"
+    perm_manager = contracts.acl.getPermissionManager(contracts.finance, convert.to_uint(Web3.keccak(text="CREATE_PAYMENTS_ROLE")))
+    contracts.acl.grantPermissionP(
+        EVM_SCRIPT_EXECUTOR,
+        contracts.finance,
+        convert.to_uint(Web3.keccak(text="CREATE_PAYMENTS_ROLE")),
+        encode_permission_params(amount_limits()),
+        {"from": perm_manager}
+    )
+
+    # Item 1
+    atc_allowed_recipients_registry = interface.AllowedRecipientRegistry("0xe07305F43B11F230EaA951002F6a55a16419B707")
+    atc_multisig_acc = accounts.at("0x9B1cebF7616f2BC73b47D226f90b01a7c9F86956", force=True)
+    atc_trusted_caller_acc = atc_multisig_acc
+    atc_top_up_evm_script_factory = interface.TopUpAllowedRecipients("0x1843Bc35d1fD15AbE1913b9f72852a79457C42Ab")
+    atc_budget_limit_after_expected = 7_000_000 * 10**18
+    atc_spend_limit_after_expected = 5_500_000 * 10**18
+
+    # Item 2
+    pml_allowed_recipients_registry = interface.AllowedRecipientRegistry("0xDFfCD3BF14796a62a804c1B16F877Cf7120379dB")
+    pml_multisig_acc = accounts.at("0x17F6b2C738a63a8D3A113a228cfd0b373244633D", force=True)
+    pml_trusted_caller_acc = pml_multisig_acc
+    pml_top_up_evm_script_factory = interface.TopUpAllowedRecipients("0x92a27C4e5e35cFEa112ACaB53851Ec70e2D99a8D")
+    pml_budget_limit_after_expected = 4_000_000 * 10**18
+    pml_spend_limit_after_expected = 3_000_000 * 10**18
+
+    # Item 3, 4
+    stonks_steth_allowed_recipients_registry = interface.AllowedRecipientRegistry("0x1a7cFA9EFB4D5BfFDE87B0FaEb1fC65d653868C0")
+    stonks_steth_multisig = accounts.at("0xa02FC823cCE0D016bD7e17ac684c9abAb2d6D647", force=True)
+    stonks_steth_trusted_caller = stonks_steth_multisig
+    stonks_steth_top_up_evm_script_factory = interface.TopUpAllowedRecipients("0x6e04aED774B7c89BB43721AcDD7D03C872a51B69")
+    stETH_token = interface.ERC20("0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84")
+    stonks_steth_contract = accounts.at("0x3e2D251275A92a8169A3B17A2C49016e2de492a7", force=True)
```

Comment:
Could you pls clarify why this contract is used?

Reply:
https://docs.lido.fi/deployed-contracts/#lido-stonks-contracts
To make a test Stonks steth transfer, you can use any steth stonks contract here